test/revolution_test.rb

```diff
@@ -0,0 +1,522 @@
+                        },
+                        "auxLights": 1,
+                        "grabber": {
+                            "motorDiagnostics": {
+                                "current": 3,
+                                "overcurret": false,
+                                "rpm": 5.5
+                            },
+                            "openClose": 1,
+                            "rotate": 1.0
+                        },
+                        "cameraHead": {
+                            "lights": 1.5,
+                            "lasers": true,
+                            "tilt": {
+                                "speed": 1.5
+                            },
+                            "tiltMotorDiagnostics": {
+                                "current": 3,
+                                "overcurret": false,
+                                "rpm": 5.5
+                            },
+                            "camera": {
+                                "focus": {
+                                    "value": 1.5
+                                },
+                                "saturation": 2,
+                                "sharpness": 2,
+                                "zoom": {
+                                    "ration": 1.5,
+                                    "speed": 1.5
+                                }
+                            }
+                        }
+                    },
+                    "578954C0A26S": {
+                        "calibrator": true,
+                        "ready": true,
+                        "distance": 15,
+                        "leak": false,
+                        "cpuTemp": 10
+                    },
+                    "63B234C0A269": {
+                        "calibrator": true,
+                        "ready": true,
+                        "distance": 15,
+                        "leak": false,
+                        "cpuTemp": 10,
+                        "battery1": {
+                            "percent": 0.8,
+                            "voltage": 1,
+                            "charging": 2
+                        },
+                        "battery2": {
+                            "percent": 0.8,
+                            "voltage": 1,
+                            "charging": 2
+                        },
+                        "motor1Diagnostics": {
+                            "current": 3,
+                            "overcurret": false,
+                            "rpm": 5.5
+                        },
+                        "motor2Diagnostics": {
+                            "current": 3,
+                            "overcurret": false,
+                            "rpm": 5.5
+                        },
+                        "speed": 5.5,
+                        "acConnected": false,
+                        "eStop": false
+                    }
+                }
+            }
+        )
```

Comment:
You could turn the JSON messages into `yml` files instead of writing it all on code, I think it would be easier to interact.